app/views/books/show.html.erb

```diff
@@ -2,3 +2,27 @@
 <p><%= @book.author %></p>
 <p><%= @book.year %> </p>
 <p><%= @book.description %></p>
+
+<br>
+<br>
+<h1>Comments</h1>
+<div class='Comments'>
+  <table>
+    <% @book.comments.each do |comment|%>
```

Comment:
As you're setting up the variable `@comments` in the controller, you can use it here instead of `@book.comments`